stdlib/in_channel.mli

```diff
@@ -123,6 +136,9 @@ val really_input_string : t -> int -> string option
     returns them in a new string.  Returns [None] if the end of file is reached
     before [len] characters have been read. *)
 
+val input_all : t -> string
+(** [input_all ic] reads all remaining data in [ic]. *)
```

Comment:
IANANES but I found the `in` a bit strange maybe `from` ?

Reply:
Sounds right, amended.